### ISSUE TITLE
Modal: Not enough contrast on disabled buttons in dialogs in dark theme

### ISSUE THREAD

**Issue by karinpb** [infor-design/enterprise#2754]
**Describe the bug**
From a usability perspective there is not enough contrast between the disabled button text. This is a usability issue. In Homepages almost all dialogs have mandatory content and thus the save button would always be disabled.

It's just too confusing to have it barely readable, it's a general usability issue.

**To Reproduce**
1. Use the dark theme
2. Access a dialog with mandatory fields that are not filled in so that the primary button is disabled

**Expected behavior**
It should be possible to see what is written on the button for normal users that does not need improved accessibility

**Version**
4.20.0

**Screenshots**
![image](https://user-images.githubusercontent.com/7961438/64510193-da182200-d2e1-11e9-933e-09d668978177.png)

**Platform**
 - OS Version: Windows 10
 - Browser Name chrome
Probably the same issue in all browsers.

**Additional context**
Asked in Teams and the comment that was made is the following:
We can review the button style if you want to report it but if its disabled. Its not required that it passes WCAG guidlines.

My standpoint is that regardless of those requirements it's a usability issue that it can be read for users without accessibility needs. It's a user experience issue as the user starts to read the dialog the primary action is an important part of the dialog and it's really strange that it is hardly visible at all.

**Comment by tmcconechy** [comment]
@kentonquatman and @kevinwhitedesign 

1. Do you think we should bring up the color on disabled modal buttons (or down) - see attached
2. What about he disabled checkbox color http://4210-beta1-enterprise.demo.design.infor.com/components/checkboxes/example-index.html?theme=soho&variant=dark&colors=2578a9 is that too light?

Let me know if you suggest new colors for those two. Thanks!

**Comment by kentonquatman** [comment]
1. For the dark theme, we should use Slate 04 instead of a dark or translucent azure.
<img width="400" alt="Screen Shot 2020-01-15 at 6 18 05 PM" src="https://user-images.githubusercontent.com/847306/72479886-ae430680-37c3-11ea-9968-84e225dd1781.png">

2. The checkmark looks good, but the disabled text could be one shade brighter and also use Slate 04.
<img width="240" alt="Screen Shot 2020-01-15 at 6 10 44 PM" src="https://user-images.githubusercontent.com/847306/72479420-61aafb80-37c2-11ea-91a5-b00a70f2007a.png">

**Comment by tmcconechy** [comment]
Just wondering it the primary button on the modal should be more closer to white (or white like the checkboxes). Idea is it is more "prominent" as it is the primary button that fires on enter so should maybe stand out more than it does? But that does solve the problem well by not making it azure/primary color?

**Comment by kentonquatman** [comment]
I don't have a preference between white or light blue on the primary button. They both look readable are WCAG AA compliant for large text. I wouldn't change the primary button unless we find a problem with it; we only know there is a problem with the disabled state.

|White Text|Blue Text|
|---|---|
|<img width="400" alt="Screen Shot 2020-01-15 at 7 19 03 PM" src="https://user-images.githubusercontent.com/847306/72482551-1695e600-37cc-11ea-9e55-def6f1b86cda.png">|<img width="400" alt="Screen Shot 2020-01-15 at 7 19 58 PM" src="https://user-images.githubusercontent.com/847306/72482552-1695e600-37cc-11ea-921a-1d09de847e5c.png">|

**Comment by tmcconechy** [comment]
Sorry my mistake - its disabled there. I agree looks good. We will take care of this.

**Comment by jbrcna** [comment]
The issue is now resolved. moving this ticket to done

**Comment by kentonquatman** [comment]
![Screen Shot 2020-02-10 at 12 59 42 PM](https://user-images.githubusercontent.com/847306/74176295-6918c580-4c05-11ea-8512-6016cf714c5c.png)

The disabled button looks much better. We also talked about fixing the validation alerts. Can we make sure the validation text is white for legibility?

**Comment by tmcconechy** [comment]
made changes to  dark mode uplift only on http://localhost:4000/components/modal/example-validation.html?theme=uplift&variant=dark

the label is bit lighter and the color text is now white by design QA. so this could use another test

**Comment by jbrcna** [comment]
changes have been made, working as expected. Tested across all browsers.
tagging this ticket as Passed and Done
https://master-enterprise.demo.design.infor.com/components/modal/example-validation.html?theme=uplift&variant=dark
v4.26.0-dev
![image](https://user-images.githubusercontent.com/56722184/74523914-0868f000-4f59-11ea-895b-d74a257e8a6a.png)
